skimage/transform/_polar.pyx

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from libc.math cimport sqrt, sin, cos, floor, ceil, M_PI
+from skimage._shared.interpolation cimport round
+
+
+def cart2pol(img, center=None, full_output=False):
+    """
+    Polar transform of an image.
+
+    Parameters
+    ----------
+
+    img : (M, N, D) ndarray
+        Input image.
+    center : (row, column) tuple or (2,) ndarray, optional
+        Center of the polar transform.
+    full_output : boolean, optional (default False)
+        Extend the output to fully enclose the image.
```

Comment:
I would add returns and a fuller description. I don't actually know what this function does.